src/components/search-wizard/QueryLayer.jsx

```diff
@@ -1,9 +1,96 @@
+  "Secure": "Yes",
+  "Geometry Type": "point",
+  "Division Heading": "Drinking Water",
+  "Layer Description": "Wells, springs and surface-water intakes used by public water systems.",
+  "Metadata Link": "https://71dvtvplh8.execute-api.us-west-2.amazonaws.com/PROD/deq-interactive-maps-docs?doc-id=PublicWaterSystemFacs_Metadata.pdf",
+  "Special Filters": "n/a",
+  "Special Filter Default To On": "N",
+  "Additional Searches": "n/a",
+  "OID Field": "",
+  "ID": "SYSFACID",
+  "NAME": "FACNAME",
+  "ADDRESS": "SYSNUMBER",
+  "CITY": "SYSNAME",
+  "TYPE": "FACTYPEDESC",
+  "Custom Symbology Field": "n/a",
+  "Legend Title": "n/a",
+  "Map Label Field": "SYSFACID",
+  "Sort Field": "n/a",
+  "Identify Attributes": "SYSFACID (System-Facility ID), SYSNUMBER (System Number), SYSNAME (System Name), FACID (Facility Identifier), FACNAME (Facility Name), FACTYPEDESC (Facility Type Description), FACTYPECODE (Facility Type Code), FACACTIVITY (Facility Activity Status), SYSTYPE (System Type), SYSACTIVITY (System Activity Status), SYSPOPULATION (System Population), SYSPOPWHSALE (System Wholesale Population), SYSPHONE (System Phone), SYSPHONEEXT (System Phone Extension), SYSADDRESS1 (System Address1), SYSADDRESS2 (System Address2), SYSCITY (System City), SYSSTATE (System State), SYSZIP (System ZIP Code)\\n",
+  "Related Tables": "None",
+  "Document Search": "https://documents.deq.utah.gov/TabsPage.aspx?AI_PageConfigID=100001&DivName=DDW",
+  "GRAMA Request": "https://deq.utah.gov/general/records-request-government-records-access-and-management-act-grama",
+  "Permit Information": "https://documents.deq.utah.gov/TabsPage.aspx?AI_PageConfigID=100123&DivName=DAQ",
+  "Additional Information": "https://documents.deq.utah.gov/TabsPage.aspx?AI_PageConfigID=100003&DivName=DDW",
+  "Comments": "",
+  "Feature Service": "",
+  "Coded Values": ""
+}
+*/
+export default function QueryLayer({
+  config,
+  selected,
+  onSelectedChange,
+  // filter,
+  // onFilterChange,
+}) {
+  const id = `query-layer-${config.index}`;
 
-  // todo - use logEvent from 'firebase/analytics' to log which layers are selected
-  return <Checkbox name={id} label={config[fieldNames.queryLayers.name]} />;
+  return (
+    <div className="my-2 flex items-center justify-between">
+      <Checkbox
+        name={id}
+        label={config[fieldNames.queryLayers.name]}
+        onChange={onSelectedChange}
+        checked={selected}
+      />
+      <Tooltip
+        trigger={
+          <a
+            href={config[fieldNames.queryLayers.metadataLink]}
+            target="_blank"
+            rel="noreferrer"
+          >
+            <Icon
+              name={Icon.Names.help}
+              className="text-gray-600"
+              label="more info"
+              size="xs"
+            />
+          </a>
+        }
+      >
+        <div className="max-w-md">
+          {config[fieldNames.queryLayers.layerDescription]}
+        </div>
+      </Tooltip>
```

Comment:
**thought**: This interaction seemed odd to me. I clicked the ? icon to find more information and it opened a new link. I expected a help icon, when clicked, to show help text. This seemed to have an external link icon behavior. 

Then I came back and slow hovered over it to find out more information about the layer. This interaction won't be accessible in a touch environment.

I wonder if the hover text and ? icon should be on the layer name and the metadata external link icon is either in the tooltip or a separate icon?

Reply:
That's a good point. I moved the hover popup to the layer name and then gave the external link a better icon.